map.ipynb

```diff
@@ -0,0 +1,386 @@
+    "    name='Fraser',\n",
+    ")\n",
+    "c = Polygon(\n",
+    "    locations=columbia_coords,\n",
+    "    color='green',\n",
+    "    name='Columbia',\n",
+    ")\n",
+    "\n",
+    "legend = LegendControl(\n",
+    "    {'Peace': 'blue', 'Fraser': 'red', 'Columbia': 'green'},\n",
+    "    name='Watersheds',\n",
+    "    position='topright',\n",
+    ")\n",
+    "\n",
+    "latitude = Text(\n",
+    "    placeholder='Enter the Latitude', description='Latitude:', disabled=False\n",
+    ")\n",
+    "\n",
+    "longitude = Text(\n",
+    "    placeholder='Enter the Longitude', description='Longitude:', disabled=False\n",
+    ")\n",
+    "\n",
+    "layer_control = LayersControl()\n",
+    "marker_group = LayerGroup()\n",
+    "\n",
+    "point_list = SelectMultiple(options=[], value=[], description='Points:', disabled=False)\n",
+    "points = []\n",
+    "\n",
+    "run = Button(\n",
+    "    description='Run',\n",
+    "    button_style='success',\n",
+    "    disabled=False,\n",
+    "    tooltip=\"Click 'Run' to start the Osprey Flask App.\",\n",
+    ")\n",
+    "run.on_click(handle_run)\n",
+    "\n",
+    "add_point = Button(\n",
+    "    description='Add Point',\n",
+    "    button_style='primary',\n",
+    "    disabled=False,\n",
+    "    tooltip='Click to add this point to the list.',\n",
+    ")\n",
+    "add_point.on_click(handle_add)\n",
+    "\n",
+    "remove_point = Button(\n",
+    "    description='Remove Point(s)',\n",
+    "    button_style='danger',\n",
+    "    disabled=False,\n",
+    "    tooltip='Click to remove selected point(s) from the list.',\n",
+    ")\n",
+    "remove_point.on_click(handle_remove)\n",
+    "\n",
+    "start_date = date_widget('Start Date:')\n",
+    "end_date = date_widget('End Date:')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "m.add_layer(p)\n",
+    "m.add_layer(f)\n",
+    "m.add_layer(c)\n",
+    "m.add_layer(marker_group)\n",
+    "\n",
+    "m += layer_control\n",
+    "\n",
+    "m.on_interaction(handle_click)\n",
+    "m.add_control(legend)\n",
+    "\n",
+    "box_layout = Layout(display='flex',\n",
+    "                flex_flow = 'column', \n",
+    "                width='110%',\n",
+    "                align_items = 'center')\n",
+    "\n",
+    "control_box = Box(children = [start_date, end_date, latitude, longitude, add_point, point_list, remove_point, run], layout=box_layout)\n",
+    "\n",
+    "AppLayout(center = m, \n",
+    "    right_sidebar = control_box,\n",
+    "    align_items = 'center')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Starts RVIC process\n",
+    "url = build_url(start_date.value, end_date.value, points)\n",
+    "input_response = requests.get(f'http://docker-dev03.pcic.uvic.ca:30111/osprey/input?{url}').content\n",
```

Comment:
Could you assign the base `http://docker-dev03.pcic.uvic.ca:30111` url to a variable so that it can just be used in f-strings whenever you make requests?